src/platform_impl/windows/window_state.rs

```diff
@@ -192,13 +189,17 @@ impl WindowFlags {
 
         let (mut style, mut style_ex) = (0, 0);
 
+        style |= WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_SYSMENU | WS_CAPTION | WS_MINIMIZEBOX;
+        style_ex |= WS_EX_ACCEPTFILES;
```

Comment:
Nit: Is there a reason that you've moved this up here? It made the changes *slightly* harder to follow.

Reply:
Since these styles are always added, it felt more intuitive to me to have them before all the checks. I don't have strong opinion about it, I can revert it if you want.

Reply:
When you put it like that, it does sound fairly reasonable.